app/frontend/courses/review/components/CoursesReview__Editor.res

```diff
@@ -399,14 +395,21 @@ let closeOverlay = (state, courseId, filter) => {
     : RescriptReactRouter.push(path)
 }
 
-let reviewNextButton = (nextSubmission, send, courseId, filter, submissionId, className) => {
+let reviewNextButton = (nextSubmission, filter, nextSubmissionId, className) => {
   ReactUtils.nullIf(
     <button
```

Comment:
I would use the `<Link>` component instead of a `button` here. `Link` components adds an `<a>` which preserves the default link behavior;